community/modules/scheduler/schedmd-slurm-gcp-v5-controller/variables.tf

```diff
@@ -103,6 +103,12 @@ variable "deployment_name" {
   type        = string
 }
 
+variable "disable_controller_public_ips" {
```

Comment:
I can't find the official docs on it anymore, but I remember tflint used to complain about negating boolean variables. In general, I agree that boolean variables should be enabling if true, disabling if false.

On the other hand, this keeps the naming the same as in v4, which is valuable. 

There's a good case for either, so I'll leave it up to you to keep it as is or invert it to "enable_controller_public_ips".

Reply:
I really wanted to keep the same exact signature as v4, even though I hate the double negation.